examples/log/src/gui.rs

```diff
@@ -77,30 +77,18 @@ impl std::io::Write for Writer {
         let timestamp = json["timestamp"].as_str().unwrap();
         let target = json["target"].as_str().unwrap();
         let msg = json["fields"]["message"].as_str().unwrap();
+        let datetime_local = timestamp
+            .parse::<chrono::DateTime<chrono::Utc>>()
+            .unwrap()
+            .with_timezone(&chrono::Local)
+            .format("%m/%d %H:%M:%S");
```

Comment:
Would you mind retaining the "%Y-%m-%dT".

So: `%Y-%m-%dT%H:%M:%S` (we can drop the `%.6fZ`)

Thanks!

Reply:
Plan to move the CLI logger into `runtime` with #520

Reply:
Changed the timestamp for now as suggested. `runtime` looks interesting. Let me know if there are any good first issues I can pick up related to `Logger`.

Reply:
I need to do a little more heavy-lifting to setup that change, however, once its done I'll create some issues 🙏